indexer/sql/src/pubsub.rs

```diff
@@ -112,16 +25,19 @@ pub enum PubSubType {
 #[cfg(test)]
 mod tests {
     use {
-        super::*, assertor::*, sqlx::postgres::PgPoolOptions, std::time::Duration,
+        super::*,
+        assertor::*,
+        sqlx::postgres::{PgListener, PgPoolOptions},
+        std::time::Duration,
         tokio_stream::StreamExt,
     };
 
     #[ignore]
     #[tokio::test]
     async fn test_postgres_pubsub() -> anyhow::Result<()> {
-        let pool = sqlx::PgPool::connect("postgres://postgres@postgres/grug_test").await?;
+        let pool = sqlx::PgPool::connect("postgres://postgres@localhost/grug_test").await?;
```

Comment:
Use ENV `DB_HOST` to allow automatic docker containers in the CI, but `localhost` when running locally on computers.